12-offer-encoding.md

```diff
@@ -214,16 +214,27 @@ The human-readable prefix for offers is `lno`.
     1. type: 16 (`offer_paths`)
     2. data:
         * [`...*blinded_path`:`paths`]
-    1. type: 18 (`offer_issuer`)
+    1. type: 18 (`offer_compact_paths`)
+    2. data:
+        * [`...*compact_blinded_path`:`paths`]
+    1. type: 20 (`offer_issuer`)
     2. data:
         * [`...*utf8`:`issuer`]
-    1. type: 20 (`offer_quantity_max`)
+    1. type: 22 (`offer_quantity_max`)
     2. data:
         * [`tu64`:`max`]
-    1. type: 22 (`offer_node_id`)
+    1. type: 24 (`offer_node_id`)
     2. data:
         * [`point`:`node_id`]
```

Comment:
I understand the desire for cleanliness, but this breaks everyone's test code.  Just make offer_compact_paths number 24 please!